webcurator-harvest-agent-h3/src/main/java/org/webcurator/harvestagent/h3/webapp/beans/config/AgentConfig.java

```diff
@@ -279,6 +278,7 @@ public HarvestCoordinatorNotifier harvestCoordinatorNotifier() {
     public DigitalAssetStore digitalAssetStore() {
         DigitalAssetStoreClient bean = new DigitalAssetStoreClient(digitalAssetStoreScheme, digitalAssetStoreHost, digitalAssetStorePort, restTemplateBuilder);
         bean.setFileUploadMode(digitalAssetStoreFileUploadMode);
+        bean.setHarvestBaseUrl(String.format("%s://%s:%d", harvestAgentScheme, harvestAgentHost, harvestAgentPort));
```

Comment:
@leefrank9527 Should this be taking into account the context as well? In case a user wants to set a context for Store or the harvest agents too?

Reply:
Yes. When the context-path PR is merged, harvestAgent.baseUrl shoud be taken here.